### PR TITLE
Add permanet current limits fields when creating Lines

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: src/translations/fr.json [hunk 1]
```diff
@@ -326,5 +326,8 @@
     "SeriesResistanceText": "Résistance série",
     "SeriesReactanceText": "Réactance série",
     "ShuntConductanceText": "Conductance parallèle",
-    "ShuntSusceptanceText": "Susceptance parallèle"
+    "ShuntSusceptanceText": "Susceptance parallèle",
+
+    "PermanentCurrentLimitText": "IMAP",
```
Comment (FranckLecuyer) [inline src/translations/fr.json:331]: IST ? to be confirmed ...

File: src/translations/fr.json [hunk 2]
```diff
@@ -326,5 +326,8 @@
...
+    "permanentCurrentLimitGreaterThanZero": "La valeur IMAP doit être strictement positive"
```
Comment (FranckLecuyer) [inline src/translations/fr.json:332]: IST ? to be confirmed ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/translations/fr.json (hunk 1 above)
Comment (sBouzols) [reply src/translations/fr.json:331]: It's IMAP in the US, will confirm this with POs...

Comment (sBouzols) [reply src/translations/fr.json:331]: Ok @ChristineHesse seems to be OK with 'IST', edited!

File: src/translations/fr.json (hunk 2 above)
Comment (sBouzols) [reply src/translations/fr.json:332]: edited

**Review by FranckLecuyer — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**